src/hyp3_sdk/hyp3.py

```diff
@@ -444,21 +449,38 @@ def submit_insar_isce_burst_job(self,
         """
         arguments = locals().copy()
         arguments.pop('self')
+        arguments.pop('args')
+
+        if len(args) == 2:
+            arguments['reference'] = args[0]
+            arguments['secondary'] = args[1]
+
+        if arguments['granule1']:
```

Comment:
@jhkennedy I believe we're already doing what you're suggesting here (allowing people to use granules but raising a warning if they do. I like raising an error if neither are specified and will add that. Also correct that we're not breaking yet, just prepping to.

Reply:
You're breaking the interface by not maintaining the same number and order of positional arguments. For example, this works with the current SDK and my suggestion but not with this PR:
 ```python
 job = hyp3.submit_insar_job('S1_...-BURST`, 'S1_...-BURST`, 'my_job_name')
 ```

Reply:
I'll just open a PR

Reply:
Thanks!